src/Keboola/ElasticsearchWriter/Mapping/ColumnsMapper.php

```diff
@@ -78,6 +81,14 @@ private function mapValue(ColumnMapping $column, string $value)
 			return (bool) $value;
 		}
 
+		if (in_array($type, self::ARRAY_TYPES, true)) {
```

Comment:
jj vypadá to dobře, ale chtělo by to na to ještě test

Reply:
Test som pridal.